builtin/read-tree.c

```diff
@@ -162,6 +162,7 @@ int cmd_read_tree(int argc, const char **argv, const char *cmd_prefix)
 	opts.head_idx = -1;
 	opts.src_index = &the_index;
 	opts.dst_index = &the_index;
+	opts.verbose_update = isatty(2);
```

Comment:
Should `opts.verbose_update` be false if `--quiet` is specified?

Reply:
It appears that `quiet` means something other than "not verbose". This is non-obvious.

The `quiet` member in `struct unpack_trees_options` only toggles the output of _error messages_. This is different than the `verbose_update` member which toggles the existing progress indicators during the checkout.

Today, if you say `git read-tree --verbose --quiet -mu HEAD`, then you will still get progress indicators during the "Checking out files" step.

Reply:
Thanks for the details here, I couldn't quite figure out what `--quiet` was controlling.